### DISCUSSION TITLE
GLB Rendered black and white

### DISCUSSION THREAD

**Discussion by GVAMDaniel** [sceneview/sceneview#516] [Q&A] [answered]
I have two questions:

I have a .glb model which when rendered in the scene appears completely black if there is not enough lighting or black and white and there is enough lighting. If I open the same model from the Gestaltor software, for example, it appears correctly with its colors. Do I have to configure something in the scene so that it looks correctly, is it a rendering engine error, should I change the rendering engine?

My other question, my model is a hole in the ground with water, the intention is that this model is rendered at the level of the ground so that it gives the sensation that there is a hole and by looking at it from above you can see the depth. Is this possible?
I have tried to modify the position of both the model and the anchor but the only thing I can do is scale the model in different ways. I have also tried to edit the model with Blender so that it would be below the X axis to see if it was a problem with the model itself, but I haven't achieved it either.

In the example of ArCore's augmented images they present an image that gives the sensation of depth but on a wall, I want to achieve the same thing but on the floor.

[https://developers.google.com/ar/develop/augmented-images](https://developers.google.com/ar/develop/augmented-images)

I would appreciate any help.

**Comment by osoikatatsumuri** [comment, marked as answer]
Met with this problem about few days ago, but my models were always black. I added this in sessionConfiguration.

```
config.lightEstimationMode =
Config.LightEstimationMode.ENVIRONMENTAL_HDR
```

**Comment by ThomasGorisse** [comment]
You can also define your own main and indirect light
